cloud-content/impl/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.sakaiproject.content</groupId>
+    <artifactId>cloud-content</artifactId>
+    <version>11-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <name>Sakai Cloud Resources Providers (impl)</name>
+  <groupId>org.sakaiproject.content</groupId>
+  <artifactId>cloud-content-impl</artifactId>
+  <packaging>jar</packaging>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
The first 3 properties should be dropped as they will be pulled in from the Sakai master pom and we want them to stay in sync so when we change them they don't have to be updated in lots of places.